### PR TITLE
Adding instructions to download Personal Desktop

### PR DESCRIPTION
**Pull request by it33**

### CODE REVIEW

**Review by jespino — CHANGES_REQUESTED** [review]
I miss here a comment about the Linux installation, looks like only windows or mac users can try it.

Can we add something like "or download the linux binary from the github project releases page: https://github.com/mattermost/focalboard/releases".

**Comment by it33** [comment]
Added!

**Comment by it33** [comment]
@jespino updated per your suggestion

**Comment by jespino** [comment]
@it33 still looks like we have no desktop-app in linux, and we have.

**Comment by it33** [comment]
@jespino sorry, missed that, added now!

File: README.md [hunk 1]
```diff
@@ -13,6 +13,22 @@ It helps define, organize, track and manage work across individuals and teams. F
...
 
+## Try out Focalboard 
+
+**Focalboard Personal Desktop**
+
+Try out **Focalboard Personal Desktop** by going to the Windows Store or the Apple AppStore, searching for `Focalboard` and installing to run the compiled version locally. 
+
+If you're running a Linux Desktop, [download the latest `focalboard-linux.tar.gz` release](https://github.com/mattermost/focalboard/releases), unpack the `.tar.gz` archive, and open `focalboard-app` from the `focalboard-app` folder.
+
+While we don't yet offer installs of **Focalboard Personal Desktop** outside of Store-based installs, it is in [consideration for the future](https://github.com/mattermost/focalboard/issues/99) (please upvote the ticket if you're interested in this addition).
```
Comment (jespino) [inline README.md:24]: I would add a "for mac and windows" to be consistent with the linux paragraph added above

**Comment by it33** [comment]
Thanks @jespino makes sense, how about now?

**Comment by jespino** [comment]
Better, but I was talking about the paragraph talking about not offering the desktop app outside app stores, that is only try for mac and windows, we offer the app in linux to download it directly.

**Comment by it33** [comment]
Ah, okay, I think I finally have it. @jespino how's this? 

This thread feels like a scene from a movie I saw once: https://www.youtube.com/watch?v=4IFUYg9IUgI
😃

**Review by jespino — APPROVED** [review]
Now looks good to me. :)

**Pull request merged**